src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -1057,6 +1194,34 @@ T trace(const SGMatrix<T>& A)
 }
 
 /**
+ * Method that computes the transpose of a matrix.
+ *
+ * @param A The matrix
+ * @return The transposed matrix
+ */
+template <typename T>
+SGMatrix<T> transpose_matrix(const SGMatrix<T>& A)
```

Comment:
no I mean that transposing a matrix (i.e. changing the order in memory) is often not necessary, but one can just mark them as transposed and the subsequent operation realises that
Nothing to do with complex adjoints